gtfs/spec/en/reference.md

```diff
@@ -76,7 +76,7 @@ Presence conditions applicable to fields and files:
 - **Float** - A floating point number.
 - **Integer** - An integer.
 - **Phone number** - A phone number.
-- **Time** - Time in the HH:MM:SS format (H:MM:SS is also accepted). The time is measured from "noon minus 12h" of the service day (effectively midnight except for days on which daylight savings time changes occur). For times occurring after midnight, enter the time as a value greater than 24:00:00 in HH:MM:SS local time for the day on which the trip schedule begins. <br> *Example: `14:30:00` for 2:30PM or `25:35:00` for 1:35AM on the next day.*
+- **Time** - Time in the HH:MM:SS format (H:MM:SS is also accepted). The time is measured from "noon minus 12h" of the service day (effectively midnight except for days on which daylight savings time changes occur). For times occurring after midnight, enter the time as a value greater than 24:00:00 in HH:MM:SS for the day on which the trip schedule begins. <br> *Example: `14:30:00` for 2:30PM or `25:35:00` for 1:35AM on the next day.*
```

Comment:
Because we're already changing this definition: I'd argue that the "for the day on which the trip schedule begins" confuses more than it helps. I think we should either a) state (again here as before) that the time is relative to the service day, or b) don't add any alternative phrase as a GTFS "Time" is *always* relative to the service day.

Reply:
@derhuerst what about the following definition?

Time - Time in the HH:MM:SS format (H:MM:SS is also accepted). The time is measured from "noon minus 12h" of the service day (effectively midnight except for days on which daylight savings time changes occur). For times occurring after midnight `on the service day`, enter the time as a value greater than 24:00:00 in HH:MM:SS. [remove "for the day on which the trip schedule begins"]

Reply:
Sounds good.